packages/edit-site/src/components/sidebar-navigation-screen-pattern/use-navigation-menu-content.js

```diff
@@ -51,6 +54,7 @@ function getBlocksOfTypeFromBlocks( targetBlockType, blocks ) {
 
 export default function useNavigationMenuContent( postType, postId ) {
 	const { record } = useEditedEntityRecord( postType, postId );
+	const selectedCoreStore = useSelect( coreStore );
```

Comment:
```suggestion
	const { getNavigationFallbackId } = useSelect( unlock(coreStore) );
```

I think we can stick with the patterns used elsewhere in the codebase here to make it clearer.

Reply:
> I had to commit with a no-verify flag because if I have a variable which is dependent on a hook, defining at the start of the function results in this linting error:

This issue is because you can't all hooks conditionally.

I think this is why you get the message. 

Accessing the selector from the store doesn't incur a (major) penalty. So just do that at the top and avoid all hooks ordering problems.